php-ext-install soap error

Hi, I need to to build a image with soap module.

When I try to launch: 'docker-php-ext-install soap' I get the following error:

In file included from /usr/src/php/ext/soap/soap.c:22:
/usr/src/php/ext/soap/php_soap.h:24:10: fatal error: ext/standard/info.h: No such file or directory
   24 | #include "ext/standard/info.h"

I tried to do the same with the php:8.1-apache image and everything works fine.

Thanks
Emiliano

--- a/apache/Dockerfile
+++ b/apache/Dockerfile
@@ -56,8 +56,6 @@ RUN set -ex; \
 	; \
 	docker-php-ext-enable imagick opcache redis; \
 	docker-php-source delete; \
-# Header files ".h"
-	rm -r /usr/local/include/php/ext; \
 	rm -r /tmp/pear; \
 # Display installed modules
 	php -m; \

